installers/olm/Makefile

```diff
@@ -66,7 +66,7 @@ tools: ## Download tools needed to build bundles
 tools: tools/$(SYSTEM)/jq
 tools/$(SYSTEM)/jq:
 	install -d '$(dir $@)'
-	curl -fSL -o '$@' "https://github.com/stedolan/jq/releases/download/jq-1.6/jq-$$(SYSTEM='$(SYSTEM)'; \
+	curl -fSL -o '$@' "https://github.com/jqlang/jq/releases/download/jq-1.6/jq-$$(SYSTEM='$(SYSTEM)'; \
```

Comment:
Where's this change come from?

Reply:
https://github.com/stedolan/jq was moved to https://github.com/jqlang/jq -- see https://github.com/jqlang/jq/blob/master/NEWS.md#17

(I meant to flag that, sorry)